pkgs/development/interpreters/python/hooks/python-shebangs-fixup-hook.sh

```diff
@@ -0,0 +1,26 @@
+# shellcheck shell=bash
+
+# Setup hook for fixing Python shebangs
+# Many python packages have shell scripts which end up using the host python interpreter.
+# This shell hook goes through the outputs and changes it from the host python interpreter to the target.
+echo "Sourcing python-shebangs-fixup-hook.sh"
+
+pythonFixShebangs() {
+    local output="$1"
+
+    for path in $(@gnugrep@/bin/grep -l -r '#!@pythonOnBuildForHost@' "$output"); do
+      sed -i 's|@pythonOnBuildForHost@|@python@|g' "$path"
+    done
+}
+
+pythonShebangsFixupPhase() {
+    echo "Executing pythonShebangsFixupHook"
+
+    for output in $(getAllOutputNames); do pythonFixShebangs "$output" done
```

Comment:
that sounds incorrect for python scripts in dev outputs, for example.

Reply:
Hmm, might be better to add a way to skip outputs and have it default to skipping `dev`.